travis/setup_dependencies_common.sh

```diff
@@ -48,13 +47,17 @@ for channel in $CONDA_CHANNELS; do
     conda config --add channels $channel
 done
 
-# Make defaults channel the highest priority install packages from there
-# if available rather than from e.g. conda-forge
-conda config --add channels defaults
+if [[ -z $CONDA_CHANNEL_PRIORITY ]]; then
+    CONDA_CHANNEL_PRIORITY=false
+else
+    # Make lowercase
+    CONDA_CHANNEL_PRIORITY=`echo $CONDA_CHANNEL_PRIORITY | awk '{print tolower($0)}'`
```

Comment:
I think we use the ``$(...)`` form for command substitution elsewhere, but this may be only as I prefer that form.